### ISSUE TITLE
More efficient and readable handling of a data object

### ISSUE THREAD

**Issue by thujone** [bibiqiqi/Quiz_App#6]
Instead of what you did here...

  let currentQuestion = questionsAndAnswers[questionNumber-1].question;
  let currentAnswerA = questionsAndAnswers[questionNumber-1].a;
  let currentAnswerB = questionsAndAnswers[questionNumber-1].b;
  let currentAnswerC = questionsAndAnswers[questionNumber-1].c;
  let currentAnswerD = questionsAndAnswers[questionNumber-1].d;

...you can just do this:

  let currentItem = questionsAndAnswers[questionNumber -1];
  $('.js-current-question').html(currentItem.question);
  $('.js-answer-a').html(currentItem.a);
  $('.js-answer-b').html(currentItem.b);
  $('.js-answer-c').html(currentItem.c);
  $('.js-answer-d').html(currentItem.d);

Notice how `questionsAndAnswers` is a burdensome name to use. For practical purposes, it's better just to assume a "question" is inclusive of its answers. Note how I just avoided the issue by using a generic name like `currentItem`. It's not a bad idea, because it's easy to read and understand.

**Comment by bibiqiqi** [comment]
yes, this makes sense as being more efficient. are those your kitties in your avatar?

**Comment by thujone** [comment]
haha, yes, Dim Sum and Takeout.
